client/src/components/Questions/QuestionsTab/QuestionsTab.spec.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import QuestionsTab from './QuestionsTab.jsx';
+import { shallow, mount } from 'enzyme';
+// import '../../../setupTests.js';
+
+describe('QuestionsTab unit tests', () => {
+  it ('Renders without crashing', () => {
+    shallow(<QuestionsTab />);
+  });
+  it("Renders 'Reviews' and 'Questions' in tab", () => {
```

Comment:
Nice basic enzyme testing. Only one thing I like to recommend here is for you to use a single quote. 💯

Reply:
I am nesting quotes so I use ' and ". :)